### PR TITLE
go: update head to 1.5beta1

### PR DESCRIPTION
**Pull request by tamird**

### CODE REVIEW

File: Library/Formula/go.rb [hunk 1]
```diff
@@ -11,8 +11,9 @@ class Go < Formula
   head "https://github.com/golang/go.git"
 
   devel do
-    url "https://github.com/golang/go.git", :revision => "72970f7f52b1f8d14f10df40098175763a556ebe"
-    version "72970f7f52b1f8d14f10df40098175763a556ebe"
+    url "https://storage.googleapis.com/golang/go1.5beta1.src.tar.gz"
+    sha1 "4ba487b7297cc96b712df0c4e65dafb603cc60a7"
```
Comment (DomT4) [inline Library/Formula/go.rb]: `sha256` for new additions, please. We'll be enforcing it on all formulae soon.

Comment (tamird) [inline Library/Formula/go.rb]: done

**Comment by DomT4** [comment]
Re https://github.com/tamird/homebrew/commit/efac52cec2d435703a6c71a4d363c7ecfe99e4ad - Is this no longer required for `HEAD` as well?

**Comment by tamird** [comment]
@DomT4 yep, just tested it

**Comment by mikemcquaid** [comment]
Thanks for your contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!

For future reference the preferred commit message format for simple devel version updates is `go 1.5beta1 (devel)`.

**Pull request closed without merging**